pkgs/development/tools/replay-io/default.nix

```diff
@@ -0,0 +1,65 @@
+{stdenv, lib, fetchurl, autoPatchelfHook, makeWrapper, libcxx, libX11, libXt, libXdamage, glib, gtk3, dbus-glib, openssl }:
+let
+    metadata = lib.importJSON ./meta.json;
+in
+stdenv.mkDerivation rec {
+    name = "replay-io";
+    version = metadata.date + "-" + metadata.revision;
+    binaryName = "replay";
+    srcs = [
+      (fetchurl metadata.replay)
+      (fetchurl metadata.recordreplay-lib)
+    ];
+    sourceRoot = ".";
+    unpackPhase = ''
+      runHook preUnpack
+
+      arr=($srcs)
+      tar xf ''${arr[0]} --strip-components=1
+      tar xzf ''${arr[1]}
+
+      runHook postUnpack
+    '';
+    buildInputs = [
+        autoPatchelfHook
+        makeWrapper
```

Comment:
at least these two actually belong to `nativeBuildInputs`: this attribute is used for each input that's actually used at build-time (e.g. `wrapProgram` from `makeWrapper` is executed while *building* the derivation rather than when running the software inside it).